src/hooks/index.tsx

```diff
@@ -0,0 +1,8 @@
+import { TypedUseSelectorHook, useDispatch, useSelector } from 'react-redux';
+
+import type { RootState, AppDispatch } from 'store';
+
+// Use throughout your app instead of plain `useDispatch` and `useSelector`
+type DispatchFunc = () => AppDispatch;
```

Comment:
Why is type a function type 🤔

Reply:
@nvminhtue I follow the instruction from [react-reduxjs](https://react-redux.js.org/tutorials/typescript-quick-start#define-typed-hooks) 😄